packages/lesswrong/server/repos/UsersRepo.ts

```diff
@@ -317,6 +317,15 @@ export default class UsersRepo extends AbstractRepo<DbUser> {
                       AND type = 'newDialogueChecks'
               )
           )
+          AND (
```

Comment:
man, it really feels like this whole thing wants to be joins (since you're already querying notifications for the previous `AND` block)... but yeah, if it works